helix-term/src/keymap.rs

```diff
@@ -202,7 +202,7 @@ impl Default for Keymaps {
             // extend_to_whole_line, crop_to_whole_line
 
 
-            key!('m') => Command::match_brackets,
+            key!('m') => Command::surround,
             // TODO: refactor into
```

Comment:
Can you kill these comments? They're outdated

Reply:
Done 👍🏾